examples/stable/foundationdb/README.md

```diff
@@ -0,0 +1,248 @@
+# FoundationDB
+
+According to the foundationDB documentation FoundationDB gives us the power of ACID
+transactions in a distributed database and has below properties
+
+* Multi-model data store
+* Easily scalable and fault tolerant
+* Industry-leading performance
+
+**Note**
+These steps are taken from the steps that are mentioned in the official
+repository of [foundationDB operator](https://github.com/foundationdb/fdb-kubernetes-operator).
+Since they don't have any any way to install the operator without
+building the operator image, we have followed the the upstream `Makefile`
+to create this README that can be used to create a simple foundationDB
+cluster.
+
+
+## Prerequisite 
+
+* Install GO on your machine, see the [Getting Started](https://golang.org/doc/install) guide for more information.
+* Install KubeBuilder and its dependencies on your machine, see [The KubeBuilder Book](https://book.kubebuilder.io/quick-start.html) for more information.
+* Set your $GOPATH.
+* You should have [Kustomize](https://github.com/kubernetes-sigs/kustomize) installed
+on you cluster.
+* Kubernetes 1.9+ with Beta APIs enabled.
+* PV support on the underlying infrastructure.
+* Kanister version 0.23.0 with `profiles.cr.kanister.io` CRD installed.
+
+# Installation
+
+We don't have the foundationDB helm chart yet to get the foundationDB installed
+on our Kubernetes cluster. So we will be manually installing foundationDB
+on our Kubernetes cluster.
+FoundationDB team recently started working on
+[an operator](https://github.com/foundationdb/fdb-kubernetes-operator),
+and we wil be using operator to install foundationDB cluster on our Kubernetes
+cluster.
+
+To get to know about how Kubernetes operators, please follow
+[this link](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+Steps that are mentioned below can be followed to install the foundationDB operator
+
+* Change your current directory to `$GOPATH/src/github.com` using the command `cd $GOPATH/src/github.com`
+and run `mkdir foundationdb` to create the directory `foundationdb`.
```

Comment:
Does it work out of $GOPATH?

Reply:
Hi @PrasadG193 
It doesnt work out of `GOPATH`